### PR TITLE
Set compiler encoding to UTF-8 in the generated build.gradle

### PR DESCRIPTION
**Pull request by gastaldi**

### CODE REVIEW

**Review by maxandersen — APPROVED** [review]
this will make gradle projects have same default encoding as the maven ones so +1 for this

File: devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/scala/build.gradle-template.ftl [hunk 1]
```diff
@@ -20,3 +20,6 @@ dependencies {
 group '${project_groupId}'
 version '${project_version}'
 
+compileScala {
+  scalaCompileOptions.encoding = 'UTF-8'
+}
```
Comment (gsmet) [inline devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/scala/build.gradle-template.ftl]: I see you added something for Scala. We don't need something for Kotlin?

**Review by gsmet — CHANGES_REQUESTED** [review]
Asked a question. Feel free to dismiss if it's a false alarm.

File: devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/scala/build.gradle-template.ftl (hunk 1 above)
Comment (gastaldi) [reply devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/scala/build.gradle-template.ftl]: According to https://stackoverflow.com/questions/45564450/how-to-set-kotlin-source-encoding-in-gradle: 

>Kotlin source files are always UTF-8 by design. There is no way to specify any other encoding.

**Review by gsmet — APPROVED** [review]

**Pull request merged**